### PR TITLE
docs: add radial graph demos

### PR DESCRIPTION
**Pull request by LonelySnowman**
##### Checklist

- [x] `npm test` passes
- [x] benchmarks are included
- [x] commit message follows commit guidelines
- [x] documents are updated

##### Description of change

相关issue：https://github.com/antvis/G2/issues/5902

### CODE REVIEW

**Comment by LonelySnowman** [comment]
image

image

image

**Comment by pearmini** [comment]
很优秀！图片链接在下面：

- https://mdn.alipayobjects.com/huamei_qa8qxu/afts/img/A*9FxwQIEcoDMAAAAAAAAAAAAADmJ7AQ/original
- https://mdn.alipayobjects.com/huamei_qa8qxu/afts/img/A*h3MESLqnM4QAAAAAAAAAAAAADmJ7AQ/original
- https://mdn.alipayobjects.com/huamei_qa8qxu/afts/img/A*T3NtSLBLc0sAAAAAAAAAAAAADmJ7AQ/original

**Comment by LonelySnowman** [comment]
连接已更新

File: site/examples/general/radial/demo/radial-stacked.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+const dv = ds.createView().source(data);
+
+dv.transform({
+  type: 'fold',
+  fields: ['小于5岁', '5至13岁', '14至17岁'], // 展开字段集
+  key: '年龄段', // key字段
+  value: '人口数量', // value字段
+  retains: ['State'], // 保留字段集，默认为除fields以外的所有字段
+});
+// 数据被加工成 { State: 'WY', 年龄段: '小于5岁', 人口数量: 25635 }
```
Comment (pearmini) [inline site/examples/general/radial/demo/radial-stacked.ts:22]: 这个地方可以用 data transform 里面的 fold 来完成：https://g2.antv.antgroup.com/spec/data/fold

**Review by pearmini — CHANGES_REQUESTED** [review]

File: site/examples/general/radial/demo/radial-stacked.ts (hunk 1 above)
Comment (LonelySnowman) [reply site/examples/general/radial/demo/radial-stacked.ts]: GET，已更新

**Review by pearmini — APPROVED** [review]

**Comment by pearmini** [comment]
感谢给 G2 贡献代码，欢迎加入 G2 开源先锋群！

**Pull request merged**